lib/linguist/heuristics.rb

```diff
@@ -491,5 +499,14 @@ def call(data)
         Language["XML"]
       end
     end
+
+    disambiguate ".user" do |data|
```

Comment:
Removed.
I found this document https://docs.microsoft.com/en-us/cpp/ide/working-with-project-properties suggesting:

> Tip
The .user file is a legacy feature and we recommend that you delete it in order to keep properties correctly grouped according to configuration/platform. 

Then it explains in `Important` quote under [this section](https://docs.microsoft.com/en-us/cpp/ide/working-with-project-properties#bkmkPropertySheets) why it's a bad idea to use such global settings per-user.

I think it will make sense to skip it as the vendor is not happy with it either :smile: